### DISCUSSION TITLE
Does this emulator have some sort of double key press feature?

### DISCUSSION THREAD

**Discussion by IMDAHBIGBOSS** [visualboyadvance-m/visualboyadvance-m#1167] [Q&A] [answered]
Part 3 of the chromebook saga. This will most likely be the last question I ask here.

Turns out the emulator recognizes shortcuts so I have been using the auto hold shortcut to play the game but it's not perfect cause i gotta time my key presses a lot. 

**_So if this emulator has some sort of double key press option and if so where can I find it?_**

what i mean by double key press is pressing a key once "Q" and then getting the output twice "QQ"

**Comment by rkitover** [comment, marked as answer]
It does not at this time.
